app/controllers/runs_controller.rb

```diff
@@ -39,8 +39,8 @@ def update
       redirect_to edit_run_path(@run), alert: "There was an error saving that data. Please try again."
     end
 
-    if params[@run.id36][:category]
-      @run.update(category: Category.find(params[@run.id36][:category]))
+    if params[@run.id36][:category][:category]
+      @run.update(category: Category.find(params[@run.id36][:category][:category]))
       redirect_to edit_run_path(@run), notice: 'Game/category updated.'
       return
     end
```

Comment:
seems like this works on the website, so if this is required to fix a test, i think the test is broken and not the website